### PR TITLE
Period header

### PR DESCRIPTION
**Pull request by eksoverzero**
Added X-Rate-Limit-Period headers.

This is in preparation for the changes in the 'reset_header' branch that addresses #2 

I will pull request that only when I can figure out a proper test for it

### CODE REVIEW

**Comment by tobidelius** [comment]
Will this be merged?

**Comment by tobidelius** [comment]
bump

File: README.md [hunk 1]
```diff
@@ -46,8 +46,9 @@ Currently, Rack::Attack::RateLimit can only be configured to return rate limit h
 
 Rate limit headers are:
 
-* 'X-RateLimit-Limit' - The total number of requests allowed.
-* 'X-RateLimit-Remaining' - The number of remaining requests. 
+* 'X-Rate-Limit-Limit' - The total number of requests allowed.
```
Comment (jbyck) [inline README.md]: What is the reason for changing the header name?  Merging this would require a major version bump.

**Comment by jbyck** [comment]
@sandelius sorry for the late reply. @eksoverzero this needs to be rebased against master, as their were changes made since the PR was made that introduced merge conflicts.

**Pull request closed without merging**